int16 id
int16 reg_address
int16 data_length
---
int16[] payload
